lib/auth/apiserver.go

```diff
@@ -675,6 +675,9 @@ func (s *APIServer) rotateExternalCertAuthority(auth ClientI, w http.ResponseWri
 	return message("ok"), nil
 }
 
+// getCertAuthorities returns all cert authorities that match the provided type.
+// Deprecated: Use teleport.v1.trust.TrustService.GetCertAuthorities instead.
```

Comment:
I've always seen it written like this.
```suggestion
// Deprecated: Use teleport.v1.trust.TrustService/GetCertAuthorities instead.
```